website/content/contribute/code/flow/index.md

```diff
@@ -83,41 +83,17 @@ USE_EXISTING_CONTROLLER=true NO_SIMULATED_PROGRESS=true USE_EXISTING_CLUSTER=tru
   should be used. E.g. in integration tests running against an existing setup a controller is already existing
   in the Kubernetes cluster and should not be started by the test as two controllers competing may influence the test.
 
-Setup default local kind cluster with a local image registry:
-```bash
-make kind
-```
-
-By default the docker setup points to the official docker.io registry. When developing
-locally and building/pushing docker containers the docker host and namespace should be changed. When 
-using a local kind setup with a local registry the environment can be pointed to it by setting the following 
-environment variables. (Please note that you also have to add an entry to your /etc/hosts file):
+To support local image registry host resolution append the following to /etc/hosts:
```

Comment:
Not clear what the issue is that you're telling the developer to address, nor is it clear to me what exactly you want him to do.

Reply:
Let me explain here and will be happy if you help phrase it.

So that we can use the local registry (which is a local docker image exposing the docker registry v2 protocol on localhost:5000), so that we can access it from within our k8s cluster we need to assign a FQDN to it, in our case we have assigned the FQDN of `kind-registry`, and so that its' associated with the localhost we need to add an entry into `/etc/hosts`.

Happy if you can take a go at phrasing it in an easy to understand way.

Thanks!

Reply:
Are you sure that we have to? There is no such requirement in https://kind.sigs.k8s.io/docs/user/local-registry/ (though it doesn't use FQDN) and we do add kind-registry to the same docker network. 

@froesef please advise.

Reply:
`localhost:5000` would work fine for docker images. But helm cannot work with a local registry that is using https. (At least the last time I checked some months ago). So the only way that worked to push both docker images and helm charts to the registry to `kind-registry:5000` (using https) As this is not a proper address that is available everywhere we have to register it manually in `/etc/hosts`.

Reply:
Got it, thanks.